app/models/user.rb

```diff
@@ -58,7 +59,7 @@ class User
   field :locked_at,       type: Time
 
   # Validations
-  validates :email, :name, presence: true
+  validates :name, presence: true
```

Comment:
Devise has its own validates on user email so if you submit the form with an invalid email it would return the same email error twice. 
I can see if its possible to remove the validation on Devise side to make the User model more complete